tests/test_deep/test_agents/test_custom.py

```diff
@@ -15,15 +15,20 @@
 
 class custom_policy(MlpPolicy):
     def __init__(self, state_dim, action_dim, **kwargs):
-        super(custom_policy, self).__init__(state_dim, action_dim, kwargs.get("hidden"))
+        super(custom_policy, self).__init__(
+            state_dim, action_dim, kwargs.get("policy_layers")
```

Comment:
Don't use `kwargs.get()` here. You can add `policy_layers` as a variable in the `__init__` itself. Same goes for all the other similar usages. Try to avoid `kwargs.get` as far as possible. Also, if you have to use them, specify a default value for sure.

Reply:
I'll change this